chat/command.go

```diff
@@ -256,6 +256,16 @@ func InitCommands(c *Commands) {
 	})
 
 	c.Add(Command{
+		Prefix: "/shrug",
+		Help:   "Raise your arms in the air",
+		Handler: func(room *Room, msg message.CommandMsg) error {
+			body := strings.TrimLeft(msg.Body(), "/shrug")
+			room.Send(message.NewPublicMsg(body + "¯\\_(ツ)_/¯", msg.From()))
```

Comment:
Just remembered: Room will only propagate public messages to everyone _but_ the sender. You should see it if you connect with a 2nd client.

Kinda cheating but could just make it an EmoteMsg.